apps/ejabberd/test/acl_SUITE.erl

```diff
@@ -0,0 +1,302 @@
+-module(acl_SUITE).
+-compile([export_all]).
+
+-include_lib("common_test/include/ct.hrl").
+-include_lib("proper/include/proper.hrl").
+-include_lib("eunit/include/eunit.hrl").
+
+
+all() ->
+    [
+     all_rule_returns_allow,
+     none_rule_returns_deny,
+     basic_access_rules,
+     compound_access_rules,
+     host_sepcific_access_rules,
+     global_host_priority,
+     all_and_none_specs,
+     invalid_spec,
+     different_specs_matching_the_same_user
+    ].
+
+all_rule_returns_allow(_Config) ->
+    given_acl_started(),
```

Comment:
I see this (`acl_started`) is  required by all test cases (as well as stringprep). Would it that bad if we move this to init_per_testcase or even to init_per_suite as I don't see `acl` nor `stringprep` stopping (which I think should happen at the end).